IdeventSQLServerTemplate/dbo/StoredProcedures/spCreateAddress.sql

```diff
@@ -5,7 +5,7 @@
 	@country nvarchar(50)
 AS
 BEGIN
-	INSERT Addresses
+	INSERT AddressModel
```

Comment:
Jeg troede vi besluttede os for at ændre modellen ikke databasen?

Reply:
Jeg har ikke ændret i noget af det endnu. Ville lige se hvordan det virkede inden jeg ændrede noget.